### PR TITLE
52338_User search product and admin view order

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52338)

## WHAT (optional)
- create search products for user
- create view orders for admin

## HOW
- use scope to filter, fake data and render order

## Evidence (Screenshot or Video)
Screenshot from 2022-08-22 13-20-48
Screenshot from 2022-08-22 13-20-43

### CODE REVIEW

**Comment by fonqhoanq** [comment]
ready

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
   def home
-    @pagy, @products = pagy Product.all.newest, items: Settings.pagy.item_9
+    if params
+      @products = Product
+      if params[:starts_with].present?
+        @products = @products.filter_by_starts_with(params[:starts_with])
+      end
+    else
+      @products = Product.all.newest
+    end
```
Comment (manhnt-0960) [inline app/controllers/static_pages_controller.rb]:
```suggestion
@products = Product.products.filter_by_starts_with(params[:starts_with]).newest
```
k cần check param vì nếu param trống nó sẽ trả về all record

File: app/views/admin/orders/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div class="content">
+  <div class="container-fluid">
+      <div class="row p-3">
+          <table class="table table-striped">
+            <thead>
```
Comment (manhnt-0960) [inline app/views/admin/orders/index.html.erb]: indent

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**